pkg/kubelet/apis/resourcemetrics/v1alpha1/config.go

```diff
@@ -33,28 +38,28 @@ var (
 		nil,
 		nil,
 		metrics.ALPHA,
-		"")
+		"1.18.0")
```

Comment:
For my own education, do you mind sharing any existing documentation around metric deprecation? From reading the [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20190404-kubernetes-control-plane-metrics-stability.md#stability-classes), I'm a little confused about whether we need to follow a deprecate cycle for `ALPHA` metrics, and if we can remove them immediately?

Reply:
According to the metrics deprecate policy, we absolutely can remove them immediately because they are `ALPHA`.  And, according to API deprecate policy, `/metrics/resource/v1alpha1`, it's an `ALPHA` API,  can be removed immediately too.

But, in my opinion, if we want to make this change smoothly, we can deprecate them first before removing them from the codebase.

So, it's up to you :)

Reply:
I could see arguments in both directions - anyone have strong opinions?

Reply:
Lets do a normal deprecation.